composer.json

```diff
@@ -8,7 +8,8 @@
         "friendsofsymfony1/swiftmailer": "^5.4.13 || ^6.2.5"
     },
     "require-dev": {
-        "psr/log": "*"
+        "psr/log": "*",
+        "friendsofsymfony1/doctrine1": "*"
```

Comment:
should we define the min-versions here?

should swiftmailer be added too?

Reply:
Swiftmailer is in required section.
What is the min version should be?

Reply:
> Swiftmailer is in required section

:see_no_evil: true :sweat_smile: 

> What is the min version should be?

I would pick the latest tagged one IMO

Reply:
The expert on submodule removal is @thirsch . Paging dr @thirsch 
https://github.com/FriendsOfSymfony1/symfony1/pull/302#discussion_r1506470471